File: MultiplicationTableGenerator.cpp
#include <iostream>
using namespace std;

int main(){
	cout<<"********************"<<endl;
	cout<<"Multiplication Table!"<<endl;
	cout<<"********************"<<endl;
	double num,prod;
	while(true){
	    cout<<"Enter The Number : ";
	    cin>>num;
	    cout<<"********************"<<endl;
	    for(int i=1;i<=10;i++){
	    	prod=i*num;
	    	cout<<num<<'x'<<i<<'='<<prod<<endl;
	    }
	    cout<<"********************"<<endl;
	}
	return 0;
}

File: MultiplicationTableGenerator.js
console.log("--------------------\n\nMultiplication Table Generator\n\n--------------------");

var number;
var count;
const readline = require("readline-sync");

run = true;
while(run)
{
    number = readline.question("Enter The Number : ");
    console.log("--------------------");
    count = readline.question(`Enter the number till which you want ${number}'s multiplication table : `);
    console.log(`--------------------\n\nMultiplication table of ${number} : \n`);
    for(i=1;i<=count;i++)
    {
      console.log(`${number} x ${i} = ${number*i}\n`);
    }
    console.log("--------------------");
}
